admin/database/database.go

```diff
@@ -373,7 +373,7 @@ type InsertOrganizationOptions struct {
 
 // UpdateOrganizationOptions defines options for updating an existing org
 type UpdateOrganizationOptions struct {
-	Name                                string `validate:"slug"`
+	Name                                string `validate:"min=2,max=40,slug"`
```

Comment:
hopefully we should not be having any org name greater than 40 characters otherwise during update it will fail

Reply:
The previous `slug` rule already had an upper limit of 40 characters, so that should not be a problem 🙂